vmdb/db/migrate/20150405141637_remove_port_config_from_container_service.rb

```diff
@@ -0,0 +1,54 @@
+class RemovePortConfigFromContainerService < ActiveRecord::Migration
+  def up
+    create_table :container_service_port_configs do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.integer    :port
+      t.string     :target_port
+      t.string     :protocol
+      t.belongs_to :container_service, :type => :bigint
+    end
+
+    copy_over_configs
+    remove_columns
+  end
+
+  def copy_over_configs
+    ContainerService.find_each do |service|
+      ContainerServicePortConfig.create(
+          :ems_ref              => "#{service[:ems_ref]}_#{service[:port]}_"\
```

Comment:
Please stop using the line continuation character (anywhere).  It's not necessary.  Two strings on two separate lines become one string automatically.  There is almost never a case you need it in Ruby

cc @abonas @simon3z